.eslintrc

```diff
@@ -0,0 +1,27 @@
+{
```

Comment:
Note: pulling over our standard `.eslintrc` directly for now - I have a ticket to look into inheriting/sharing configs here the same way we currently do for rubocop via TeamsharesRails.